openedx/core/djangoapps/user_authn/views/tests/test_views.py

```diff
@@ -75,12 +76,27 @@ class UserAccountUpdateTest(CacheIsolationTestCase, UrlResetMixin):
 
     ENABLED_CACHES = ['default']
 
+    def _create_account(self, username, password, email):
```

Comment:
Again, it isn't obvious why we can't just reuse CreateAccountMixin?

Reply:
See my comment above.